integration/index.ts

```diff
@@ -33,31 +69,39 @@ const grill = {
     iframe.style.width = '100%'
     iframe.style.height = '100%'
 
-    let baseUrl = `https://grill.chat/${mergedConfig.spaceId}`
-    const directOpenChatId = mergedConfig.openChatRoomDirectly?.chatId
-    if (directOpenChatId) {
-      baseUrl += `/${directOpenChatId}`
+    let baseUrl = `https://grill.chat/${mergedConfig.hub.id}`
+    const channelConfig = mergedConfig.channel
+    if (channelConfig) {
+      baseUrl += `/${channelConfig.id}`
     }
 
-    const query = new URLSearchParams()
+    const query = new QueryParamsBuilder()
+
     if (mergedConfig.order) query.set('order', mergedConfig.order.join(','))
```

Comment:
Alex proposed remvoe order for now, because it works bad, and dont want to spend additional time for it

Reply:
bad because it jumps around at first, or any other reason?

Reply:
should I just remove it from readme?

Reply:
Yes, for now lets remove from readme

Reply:
> reason



> bad because it jumps around at first, or any other reason?

In general for UX, because if we have some interesting topics but you doesnt have tokens it isnt sorted by new messages

So need to clarify UX part before global using

Reply:
ah okay got it